templates/includes/footer.html

```diff
@@ -1,9 +1,6 @@
 <footer role="contentinfo">
 	<div class="wrapper">
 
-    Built <a href="https://github.com/18F/pulse">in the open</a> as a collaboration between <a href="https://18f.gsa.gov">18F</a> and the <a href="http://gsa.gov/governmentwidepolicy">Office of Government-wide Policy</a> at the <a href="http://gsa.gov">General Services Administration</a>.
-
-    <p>SSL Labs grades are used with permission, and powered by the <a href="https://www.ssllabs.com/">Qualys SSL Labs API</a>.
-    </p>
+    Built <a href="https://github.com/18F/pulse">in the open</a> as a collaboration between <a href="https://18f.gsa.gov">18F</a> and the <a href="https://www.gsa.gov/governmentwidepolicy">Office of Government-wide Policy</a> at the <a href="https://www.gsa.gov">General Services Administration</a>.
```

Comment:
Since we're now no longer using SSL Labs, I removed the SSL Labs citation from the bottom of the site.